packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -149,27 +150,47 @@ IconSwitcher.displayName = 'IconSwitcher';
 type MenuExpanderProps = {
   isVisible: boolean;
   onClick: MouseEventHandler<HTMLDivElement>;
+  isMenuOpen: boolean;
 };
-const MenuExpander = (props: MenuExpanderProps) => (
-  <li
-    key="expander"
-    className={classnames(styles['list-item'], styles.expander, {
-      [styles.hidden]: !props.isVisible,
-    })}
-  >
-    <div
-      onClick={props.onClick}
-      className={styles['expand-icon']}
-      data-testid="menu-expander"
+const MenuExpander = (props: MenuExpanderProps) => {
+  const gtmTracking = useContext(GtmContext);
+  return (
+    <li
+      key="expander"
+      className={classnames(styles['list-item'], styles.expander, {
+        [styles.hidden]: !props.isVisible,
+      })}
     >
-      {/*
-        FIXME: define hover effect.
-        https://github.com/commercetools/merchant-center-frontend/issues/2216
-      */}
-      <BackIcon color="surface" size="big" />
-    </div>
-  </li>
-);
+      <div
+        onClick={(event) => {
+          // Since the menu is currently open, we track when the button is clicked to close the menu
+          if (props.isMenuOpen) {
+            gtmTracking.track(
+              trackingEvents.collapseMainNav.action,
+              trackingEvents.collapseMainNav.category,
+              trackingEvents.collapseMainNav.label
+            );
+          } else {
+            gtmTracking.track(
+              trackingEvents.expandMainNav.action,
+              trackingEvents.expandMainNav.category,
+              trackingEvents.expandMainNav.label
+            );
+          }
```

Comment:
nit: totally a personal opinion but I wanted to share this alternative in case you find it useful.

```suggestion
const { action, category, label } = trackingEvents[props.isMenuOpen ? 'collapseMainNav' : 'expandMainNav'];
gtmTracking.track(action, category, label);
```